crowbar_framework/config/routes.rb

```diff
@@ -135,10 +135,35 @@
   get "crowbar/:barclamp(.:format)", action: "versions", controller: "barclamp"
   post "crowbar/:barclamp/1.0/:action/:id(.:format)", controller: "barclamp"
 
-  resource :installer, only: [:show] do
-    collection do
-      get :status
-      post :start
+  scope :installer do
+    root to: "installer#index",
+      as: "installer_root"
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.